x/gamm/twap/types/expected_interfaces.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// AmmInterface is the functionality needed from a given pool ID, in order to maintain records and serve TWAPs.
+type AmmInterface interface {
```

Comment:
just a suggestion :)

```suggestion
type AMMInterface interface {
```